Documentation/admin/user-management.md

```diff
@@ -52,7 +52,9 @@ Static users are those defined directly in the Identity `ConfigMap`. Static user
       userID: "1d55c7c4-a76d-4d74-a257-31170f2c4845"
 ```
 
-A Bcrypt encoded hash of the user's password can be generated using the using the [coreos/bcrypt-tool](https://github.com/coreos/bcrypt-tool/releases/tag/v1.0.0).
+A bcrypt encoded hash of the user's password can be generated using the using the [coreos/bcrypt-tool](https://github.com/coreos/bcrypt-tool/releases/tag/v1.0.0).
+
+When generating Tectonic Console passwords with `bcrypt-tool`, using values higher than the default of `-cost=10` may result in timeouts.
```

Comment:
maybe we can mention the limit in the length as well, or don't need? I'm mentioning this because I had an issue with that and doing the bug investigation I've discovered the length limit.
for information:
https://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length